ui/service.go

```diff
@@ -95,33 +105,23 @@ func (gs *uiService) Index(ctx context.Context, token string) ([]byte, error) {
 }
 
 func (gs *uiService) CreateThings(ctx context.Context, token string, things ...sdk.Thing) ([]byte, error) {
-
 	for i := range things {
-		_, err := gs.sdk.CreateThing(things[i], "123")
+		_, err := gs.sdk.CreateThing(things[i], token)
 		if err != nil {
 			return []byte{}, err
 		}
 	}
 
-	return gs.ListThings(ctx, "123")
+	return gs.ListThings(ctx, token)
 }
 
 func (gs *uiService) ListThings(ctx context.Context, token string) ([]byte, error) {
-	tpl := template.New("things")
-	tpl = tpl.Funcs(template.FuncMap{
-		"toJSON": func(data map[string]interface{}) string {
-			ret, _ := json.Marshal(data)
-			return string(ret)
-		},
-	})
-	var err error
-
-	tpl, err = tpl.ParseGlob(templateDir + "/*")
+	tpl, err := ParseTemplate("/things.html")
```

Comment:
No need for `/`, this should be handled in the `ParseTemplate()`. Just use file name, it will be simpler.

Reply:
{"level":"warn","message":"Method index took 255.486µs to complete with error: open ui/web/templateindex.html: no such file or directory.","ts":"2021-12-09T13:24:28.490807676Z"}